src/App.spec.ts

```diff
@@ -338,22 +498,22 @@ describe('App', () => {
       }
 
       describe('routing', () => {
-
         function createReceiverEvents(): ReceiverEvent[] {
           return [
             { // IncomingEventType.Event (app.event)
+              ...baseEvent,
               body: {
                 event: {},
               },
-              ack: noop,
             },
             { // IncomingEventType.Command (app.command)
+              ...baseEvent,
               body: {
-                command: '/COMMAND_NAME',
+                command: { command: '/COMMAND_NAME' },
```

Comment:
If I'm not wrong, this is the same to https://github.com/slackapi/bolt/pull/375#discussion_r374470059

Reply:
It is, the prs are based on each other. Rebasing on your merge and this should go away. 👍